moleculekit/tools/crystalpacking.py

```diff
@@ -36,39 +41,66 @@ def _is_outside(center, cellloc):
     Creates the attributes:
            -> cellloc: location of the molecule object within the Unit Cell.
     Returns True if the molecule object is outside the Unit Cell, else
-    returns False.'''
+    returns False.
+    """
     return np.sum(np.abs(cellloc)) > 0, cellloc, center
 
 
 def _move_inside(mol, axes, cellloc, center):
-    '''Translates an HTMD molecule object inside another Unit Cell.
+    """
+    Translates an HTMD molecule object inside another Unit Cell.
     Takes as argument the vectors defining the Unit Cell the molecule
     object is currently in and translates it inside another Unit Cell.
-    '''
+    """
     neworigin = np.multiply(axes.transpose(), cellloc).transpose()  # computes translation to be applied to copy
     neworigin = np.sum(neworigin, axis=0)
     newcenter = center - neworigin  # computes difference between copy center and origin of its unit cell
     mol.center(loc=newcenter)  # places copy in the target unit cell (at an equivalent position to that of original unit cell)
 
 
 def _place_crystal(mol, size, angles, axes):
-    '''Places MoleculeCopy object inside crystal.
-    Uses methods 'is_outside' and 'move_inside'.'''
+    """
+    Places MoleculeCopy object inside crystal.
+    Uses methods 'is_outside' and 'move_inside'.
+    """
     center = np.mean(mol.get('coords'), axis=0)
     cellloc = _get_cellloc(center, angles, size)  # integer from division for each axis = unit cell identifier
     if _is_outside(center, cellloc):
         _move_inside(mol, axes, cellloc, center)
 
 
-def viewCrystalPacking(mol, hexagonal=False, style_display='NewCartoon', viewerhandle=None):
-    """ Views the crystal packing of a protein
+def generateCrystalPacking(pdbid, hexagonal=False, visualize=False, viewerhandle=None):
+    """
+    Generates the crystal packing of a PDB protein. It is possible to inspect it immediately with the visualize option.
```

Comment:
For example here put: Generates the crystal packing of a PDB protein on the first line, leave an empty line and then put the rest.

Reply:
Done.